test/test.bindings.js

```diff
@@ -8,23 +8,27 @@ describe("Equality", function() {
 
   it("should be falsy when trying to compare a NoCLObject with a scalar", function () {
     U.withContext(function (ctx) {
-      assert.equal(ctx.equals(5), false, "should be false");
```

Comment:
looks like TODOs?

Reply:
I don't remember what happened here, I suspect the test was failing and I didn't find no references of `equals` on the `ctx`, neither I remember what exactly is `ctx.equals()` I don't have clear picture what relation `ctx` has to `equals` why we'd ever want to compare things like that? Why not just `ctx1 === ctx2` or `ctx1 === 5` which would naturally be false.